Dockerfile.tests

```diff
@@ -1,12 +1,14 @@
-FROM registry.fedoraproject.org/fedora:29
+FROM docker.io/fedora:29
```

Comment:
oh my, that's no longer supported

Reply:
I know what is the reason why I created this pull request :-)

this change is explained in commit title
https://github.com/user-cont/colin/commit/6da0302fa64573312ea871aa4eed6067632e53eb.patch

f29 is the latest on which still reasonable support ostree (Or I would have to temporary switch to centos7)

I plan to drop support for ostree anyway and move to new distro as well.
But that will be done in different PR.